src/Log/logListener.ts

```diff
@@ -63,13 +71,23 @@ const addLogEntriesToStore = (dispatch: TDispatch) => () => {
 function startListening(dispatch: TDispatch) {
     sendInitialMessage();
 
+    setLogLevel(getDeviceLibContext(), 'NRFDL_LOG_CRITICAL');
+    const taskID = startLogEvents(
+        getDeviceLibContext(),
+        err => {
+            if (err) logger.info(err.message);
+        },
+        event => logger.debug(formatNrfdlLogs(event.message))
```

Comment:
If I understand this correctly, here you always use log level `debug`. Would be better, if you use a log level fitting the log level of the log message. E.g. for `NRFDL_LOG_TRACE` and `NRFDL_LOG_DEBUG` use logger.debug, for `NRFDL_LOG_INFO` use `debug.info` etc.

Reply:
Mapping the levels now.